src/test/java/io/xpring/xrpl/helpers/XrpTestUtils.java

```diff
@@ -43,4 +55,77 @@
     }
     return payments;
   }
+
+  /**
+   * Generates a random wallet and funds using the XRPL Testnet faucet.
+   */
+  public static Wallet randomWalletFromFaucet() throws XrpException, IOException, InterruptedException {
+    final Integer timeoutInSeconds = 20;
+
+    Wallet wallet = Wallet.generateRandomWallet(true).getWallet();
+    String address = wallet.getAddress();
+    final String classicAddress = Utils.decodeXAddress(address).address();
+
+    String rippledUrl = "test.xrp.xpring.io:50051";
+    XrpClient xrpClient = new XrpClient(rippledUrl, XrplNetwork.TEST);
+
+    // Balance prior to asking for more funds
+    BigInteger startingBalance;
+    try {
+      startingBalance = xrpClient.getBalance(address);
+    } catch (Exception exception) {
+      startingBalance = new BigInteger("0");
+    }
+    // Ask the faucet to send funds to the given address
+    String faucetUrl = "https://faucet.altnet.rippletest.net/accounts";
+    URL url = new URL(faucetUrl);
+    HttpURLConnection con = (HttpURLConnection)url.openConnection();
```

Comment:
optional: Generally prefer to not abbreviate variables unless you really think it's a common abbreviation.